spec/request/organizations_spec.rb

```diff
@@ -1467,6 +1468,31 @@ module VCAP::CloudController
           end
         end
 
+        it_behaves_like 'list query endpoint' do
```

Comment:
The two `it_behaves_like 'list query endpoint'` should be wrapped in two different contexts; otherwise they cannot be distinguished.

Reply:
perhaps by changing the name to something like `it behaves_like 'list query endpoint'`?